components/uid.rst

```diff
@@ -27,48 +27,95 @@ Generating UUIDs
 
-    use Symfony\Component\Uid\Uuid;
+**UUID v1**
+    Generates the UUID using the MAC address of your device and a timestamp
+    (`read UUIDv1 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-1>`__).
+    Both are obtained automatically, so you don't have to pass any constructor argument::
+
+        use Symfony\Component\Uid\Uuid;
+
+        // $uuid is an instance of Symfony\Component\Uid\UuidV1
+        $uuid = Uuid::v1();
+
+    .. tip::
+
+        It's recommended to use UUIDv7 instead of UUIDv1 because it provides
+        better entropy.
+
+**UUID v3**
+    Generates UUIDs from names that belong, and are unique within, some given namespace
+    (`read UUIDv3 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-3>`__).
+    This variant is useful to generate UUIDs from arbitrary strings. It works by
+    populating the UUID contents with the``md5`` hash of concatenating the namespace
+    and the name::
+
+        use Symfony\Component\Uid\Uuid;
+
+        $namespace = Uuid::fromString(Uuid::NAMESPACE_OID);
+        // $name can be any arbitrary string
+        // $uuid is an instance of Symfony\Component\Uid\UuidV3
+        $uuid = Uuid::v3($namespace, $name);
+
+    The namespace can be any valid UUID, such as a random value generated with
+    ``Uuid::v4()`` but the UUID spec provides some default namespaces:
+
+      * ``Uuid::NAMESPACE_DNS`` if you are generating UUIDs for `DNS entries <https://en.wikipedia.org/wiki/Domain_Name_System>`__
+      * ``Uuid::NAMESPACE_URL`` if you are generating UUIDs for `URLs <https://en.wikipedia.org/wiki/URL>`__
+      * ``Uuid::NAMESPACE_OID`` if you are generating UUIDs for `OIDs (object identifiers) <https://en.wikipedia.org/wiki/Object_identifier>`__
+      * ``Uuid::NAMESPACE_X500`` if you are generating UUIDs for `X500 DNs (distinguished names) <https://en.wikipedia.org/wiki/X.500>`__
+
+**UUID v4**
+    Generates a random UUID (`read UUIDv4 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-4>`__),
+    so you don't have to pass any constructor argument::
+
+        use Symfony\Component\Uid\Uuid;
+
+        // $uuid is an instance of Symfony\Component\Uid\UuidV4
+        $uuid = Uuid::v4();
+
+**UUID v5**
+    It's the same as UUIDv3 (explained above) but it uses ``sha1`` instead of
+    ``md5`` to hash the given namespace and name
+    (`read UUIDv5 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-5>`__).
+
+**UUID v6**
+    It's lexicographically sortable (like :ref:`ULIDs <ulid>`) and contains a 60-bit timestamp and 63 extra unique bits
+    (`read UUIDv6 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-6>`__)::
+
+        use Symfony\Component\Uid\Uuid;
+
+        // $uuid is an instance of Symfony\Component\Uid\UuidV6
+        $uuid = Uuid::v6();
+
+    .. tip::
```

Comment:
Duplicate?

Reply:
Yes, it was a copy+paste mistake. The tip here should say "v6" instead of "v1". Fixed. Thanks.